numba/cuda/dispatcher.py

```diff
@@ -358,6 +358,18 @@ def _prepare_args(self, ty, val, stream, retr, kernelargs):
             kernelargs.append(ctypes.c_double(val.real))
             kernelargs.append(ctypes.c_double(val.imag))
 
+        elif isinstance(ty, types.EnumMember):
```

Comment:
Can we move this beneath the `Record` and `BaseTuple` cases? `_prepare_args` is the slowest part of the launch and my concern is that each case adds a measurable impact on the launch time - if we move this check last it should avoid the impact on existing types.